src/MyTested.AspNetCore.Mvc.ModelState/Builders/Contracts/Models/IAndModelStateBuilder.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace MyTested.AspNetCore.Mvc.Builders.Contracts.Models
+{
+    public interface IAndModelStateBuilder : IModelStateBuilder
```

Comment:
No documentation summary on this interface. I know it is boring but the code should be as consistent as possible. Just copy it from another IAnd... interface.